interfaces/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(pal_interfaces_h_files
     inc/c_pal/arithmetic.h
     inc/c_pal/execution_engine.h
     inc/c_pal/async_socket.h
+    inc/c_pal/file_util_linux.h

```

Comment:
>file_util_linux [](http://example.com/codeflow?start=14&length=15)

Typically we have a PR with at least a (mostly) empty .c file that includes the new headers also so we know that the code in the header will compile.

Reply:
I will do that next time!